### ISSUE TITLE
Update COVID-19 FAQ with new CLC Visitation Policy 

### ISSUE THREAD

**Issue by lalexanderson-dsva** [department-of-veterans-affairs/va.gov-team#21631]
## Issue Description
A new VHA policy is out updating visitation policies w/r/t CLCs and SCLCs. We need to review the policy to identify what we can say at a national level (applicable to all) and update the COVID-19 FAQ accordingly. 

---
## Tasks
- [x] _Review the new strategy & identify what needs to be updated. (Danielle/Randi)_
- [ ] _Draft update to FAQ on visitation with updated content. (Randi)_
- [ ] _Run update by Dr. Mole. (Lauren)_
- [ ] _Get content approved by JTF. (Lauren)_
- [ ] _Hand off approved content for implementation (Lauren)_
- [x] _Update FAQ English page with JTF content. (Randi, Selina)_
- [x] _Update content in chatbot. (Selina, Randi)_
- [x] _Send updates to translation. (Randi)_
- [x] _Update content in Spanish and Tagalog pages (Selina, Randi)_

## Acceptance Criteria
- [x] _Content is updated on FAQ page in all languages & in mini hub. _
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by RLHecht** [comment]
@lalexanderson-dsva @DanielleThierryUSDSVA I've reordered the questions that we discussed in Slack. I added a new first sentence to the visitation answer and I created an anchor link at the bottom of the first appointments question to go to the accordions. I put the schedule or cancel appointments question in the same place that the visitation one was in the accordions but let me know if you think it should be moved up. preview-prod.vfs.va.gov/preview?nodeId=3014

**Comment by RLHecht** [comment]
This page is now published with the updated content. @lunascoop Can you please get the Chatbot work started for this?

**Comment by lunascoop** [comment]
@RLHecht chatbot PR ready for review.

**Comment by RLHecht** [comment]
Copy for translation sent 3/18

**Comment by RLHecht** [comment]
@lunascoop Here is the Tagalog copy for this page: [https://app.zenhub.com/files/133843125/51c35446-7762-4354-9167-4356659830cb/download](https://app.zenhub.com/files/133843125/51c35446-7762-4354-9167-4356659830cb/download)

**Comment by RLHecht** [comment]
@lunascoop Here are the translations back for these updates. 

[https://app.zenhub.com/files/133843125/d60cc4d6-bfc6-4cb0-ba78-548ca069349e/download](https://app.zenhub.com/files/133843125/d60cc4d6-bfc6-4cb0-ba78-548ca069349e/download)

[https://app.zenhub.com/files/133843125/cd65e4d6-093a-48f0-83b6-63de9060fa7d/download](https://app.zenhub.com/files/133843125/cd65e4d6-093a-48f0-83b6-63de9060fa7d/download)

**Comment by lunascoop** [comment]
@RLHecht These pages are ready for review. It seems that the Spanish version was a bit more more outdated and required more changes. I've included the changes I made in the comments of the screenshot.

Tagalog: http://preview-prod.vfs.va.gov/preview?nodeId=16475
Spanish: http://preview-prod.vfs.va.gov/preview?nodeId=14580

<img width="1071" alt="Screen Shot 2021-03-22 at 3 38 45 PM" src="https://user-images.githubusercontent.com/42546515/112051201-08f29380-8b28-11eb-9a63-9429e74dfe1b.png">
